Makefile

```diff
@@ -12,14 +12,15 @@ MDBOOK_BIN := $(TOOLS_BIN_DIR)/mdbook
 export PATH := $(PATH):$(TOOLS_BIN_DIR)
 
 ## ------------------------------------
-## Resolve placeholders as tags
+## mdbook plugins
 ## ------------------------------------
 RELEASETAGS := $(TOOLS_BIN_DIR)/mdbook-releasetags
 $(RELEASETAGS): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go build -tags=tools -o $(TOOLS_BIN_DIR)/mdbook-releasetags ./releasetags
 
-.PHONY: releasetags
-releasetags: $(RELEASETAGS)
```

Comment:
I removed the phony for releasetags since it seemed unnecessary to me. We were not using it anywhere so the only thing it could be used for is to manually build the binary releasetags.